Tita-Clone/Tita-Clone/Views/TextFieldView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TextField.swift
+//  Tita-Clone
+//
+//  Created by 혜인 on 2021/12/08.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class TextFieldView: UIView, UITextFieldDelegate{
+    lazy var textField = UITextField().then {
```

Comment:
```suggestion
class TextFieldView: UIView, UITextFieldDelegate{
    private let viewBounds = UIScreen.main.bounds
    lazy var textField = UITextField().then {
```
레이아웃 작성에 필요한 `bounds`를 선언해서 사용하세요